torch/_torch_docs.py

```diff
@@ -3355,11 +3355,21 @@ def parse_kwargs(desc):
 .. math::
     c = (u u^T)^{-1} b
 
-.. note:: :attr:`b` is always a 2-D tensor, use `b.unsqueeze(1)` to convert a vector.
+`torch.potrs(b, u)` can take in 2D inputs `b, u` or inputs that are
+batches of 2D matrices. If the inputs are batches, then returns
+batched outputs `c`
+
+.. note::
+
+    The :attr:`out` keyword only supports 2D matrix inputs, that is,
+    `b, u` must be 2D matrices.
 
 Args:
-    b (Tensor): the right hand side 2-D tensor
-    u (Tensor): the input 2-D tensor, a upper or lower triangular Cholesky factor
+    b (Tensor): input matrix of size :math:`(*, m, k)`,
+                where :math:`*` is zero or more batch dimensions
+    u (Tensor): input matrix of size :math:`(*, m, m)`,
+                where :math:`*` is zero of more batch dimensions composed of
+                upper or lower triangular Cholesky factor
```

Comment:
What is a "Cholesky factor"? Is that common terminology?

Reply:
Cholesky factors are those that arise from the Cholesky decomposition / factorization. I think it's well understood in this context.